modules/serverless-upgrade-sub-channel.adoc

```diff
@@ -5,22 +5,22 @@
 [id="serverless-upgrade-sub-channel_{context}"]
 = Upgrading the Subscription Channel
 
-In {ServerlessProductName} version 1.6.0, the only available Subscription Update Channel was `preview-4.3`.
-To upgrade from 1.6.0 to the latest version, you must update the channel to `4.4`.
+To upgrade to the latest version of {ServerlessProductName}, you must update the Operator subscription channel to `4.5`.
 
-If you are upgrading from {ServerlessProductName} version 1.5.0, or earlier, to version 1.7.0, you must complete the following steps:
+If you are upgrading from {ServerlessProductName} version 1.5.0, or earlier, to version 1.7.1, you must complete the following steps:
 
 * Upgrade to {ServerlessProductName} version 1.5.0, by selecting the `techpreview` channel.
 * After you have upgraded to 1.5.0, upgrade to 1.6.0 by selecting the `preview-4.3` channel.
-* Finally, after you have upgraded to 1.6.0, upgrade to the latest version by selecting the `4.4` channel.
+* After you have upgraded to 1.6.0, upgrade to 1.7.0 by selecting the `4.4` channel.
+* Finally, after you have upgraded to 1.7.0, upgrade to the latest version by selecting the `4.5` channel.
 
 [IMPORTANT]
 ====
 After each channel change, wait for the pods in the `knative-serving` namespace to get upgraded before changing the channel again.
 ====
 
 .Prerequisites
-* You have installed a Technology Preview version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
+* You have installed a previous version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
```

Comment:
Is _Automatic_ capitalized?

Reply:
Yeah, it comes from the actual UI https://docs.openshift.com/container-platform/4.4/operators/olm-adding-operators-to-cluster.html#olm-installing-operators-from-operatorhub_olm-adding-operators-to-a-cluster :slightly_smiling_face: